doc/sphinx-guides/source/api/changelog.rst

```diff
@@ -12,6 +12,8 @@ v6.2
 
 - **/api/datasets/{id}/versions/{versionId}**: The includeFiles parameter has been renamed to excludeFiles. The default behavior remains the same, which is to include files. However, when excludeFiles is set to true, the files will be excluded. A bug that caused the API to only return a deaccessioned dataset if the user had edit privileges has been fixed.
 - **/api/datasets/{id}/versions**: The includeFiles parameter has been renamed to excludeFiles. The default behavior remains the same, which is to include files. However, when excludeFiles is set to true, the files will be excluded.
+- **/api/info/openapi/{format}** This new endpoint will provide the OpenAPI specification for the specified format "json" or "yaml". 
```

Comment:
For a short time, before releasing 6.1 we starting adding every new endpoint and change to this changelog.

Then we decided it's too much. It's a maintenance burden. So now it's only breaking changes (see the text at the top).

So pleas remove this.

As I said [elsewhere](https://github.com/IQSS/dataverse/pull/10328/files#r1492911209), I could maybe see adding "/openapi is broken" under v6.0. That was a breaking change. I could also see flling in changes from v5.whatever as well.